jlm/tooling/CommandLine.cpp

```diff
@@ -879,21 +892,44 @@ JlmOptCommandLineParser::ParseCommandLineArguments(int argc, char ** argv)
               "Write theta-gamma inversion statistics to file.")),
       cl::desc("Write statistics"));
 
+  auto llvmInputFormat = JlmOptCommandLineOptions::InputFormat::Llvm;
+  auto mlirInputFormat = JlmOptCommandLineOptions::InputFormat::Mlir;
+
+  cl::opt<JlmOptCommandLineOptions::InputFormat> inputFormat(
+      "input-format",
+      cl::desc("Select input format:"),
+      cl::values(
+          ::clEnumValN(
+              llvmInputFormat,
+              JlmOptCommandLineOptions::ToCommandLineArgument(llvmInputFormat),
+              "Input LLVM IR [default]"),
+          ::clEnumValN(
+              mlirInputFormat,
+              JlmOptCommandLineOptions::ToCommandLineArgument(mlirInputFormat),
+              "Input MLIR")),
+      cl::init(llvmInputFormat));
+
```

Comment:
I don't know about LLVM (.ll seems commonly used), but we could define our own for MLIR RVSDG, which I was thinking would simply be '.rvsdg'.

Reply:
I believe `.rvsdg` is not a good ending, because it does not indicate what kind of RVSDG it is. I would have more thought about `.mlir` or something like that.

Reply:
There is currently no way of printing RVSDG IR, and if we have the option to generate MLIR RVSDG would it then make sense to create our own IR printer?
.mlir could be any type of dialect, so this is not very descriptive.